### ISSUE TITLE
mod_security false positives breaking fillable forms and email functions in wordpress

### ISSUE THREAD

**Issue by m0be1** [quangdungtn90/timthumb#10]
I cant seem to figure out how exactly to make exceptions to rules. I am new to mod_security. 
I do not want to disable the rules but the following has failed. All iterations have failed via cli, so I am doing something wrong here. Does this line below belong somewhere in http.conf ?  

SecRuleUpdateTargetById 981173 !ARGS:gform_ajax:
--bash  !ARGS event not found

error from logs: 

ModSecurity: Access denied with code 403 (phase 2). Pattern match "([\\\\~\\\\!\\\\@\\\\#\\\\$\\\\%\\\\^\\\\&\\\\*\\\\(\\\\)\\\\-\\\\+\\\\=\\\\{\\\\}\\\\[\\\\]\\\\|\\\\:\\\\;\\"\\\\'\\\\\\xc2\\xb4\\\\\\xe2\\x80\\x99\\\\\\xe2\\x80\\x98\\\\`\\\\<\\\\>].*?){5,}" at ARGS:gform_ajax. [file "/etc/httpd/crs/owasp-modsecurity-crs/base_rules/modsecurity_crs_41_sql_injection_attacks.conf"] [line "159"] **[id "981173"**] [rev "2"] [msg "Restricted SQL Character Anomaly Detection Alert - Total # of special characters exceeded"] [data "Matched Data: & found within ARGS:gform_ajax: form_id=2&title=&description=&tabindex=1"] [ver "OWASP_CRS/2.2.9"] [maturity "9"] [accuracy "8"] [tag "OWASP_CRS/WEB_ATTACK/SQL_INJECTION"]
